test-scripts/tomcat-deployment/infra-provision.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+# ----------------------------------------------------------------------------
+#
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Same as the above comment

Reply:
Fixed with 1479a08a77d83ce3fe12f93a1f691490f7e8e860